eng/Versions.props

```diff
@@ -58,6 +58,7 @@
       https://vside.myget.org/F/vs-impl/api/v3/index.json;
       https://myget.org/F/roslyn_concord/api/v3/index.json;
       https://vside.myget.org/F/devcore/api/v3/index.json;
+      C:\roslyn\artifacts\packages\Debug\Release;
```

Comment:
This doesn't look correct

Reply:
It isn't and it will go away when the PR is ready. I did the exact same thing in the Round 1 shim PR.

Reply:
Thx